src/app/src/components/annotations/annotator.tsx

```diff
@@ -520,7 +525,7 @@ export default class Annotator extends Component<
     this.setState({ annotatedAssetsHidden: flag });
   }
 
-  private getInference() {
+  private getInference(reanalyse: boolean) {
```

Comment:
To prevent a ripple-on effect due to the cache change across the codebase, set a default `true`

```javascript
private getInference(reanalyse = true) {...}
```

So that you don't have to change the call signatures in the sidebar etc and only your cache retrieve functions need to provide this additional variable due to change

Reply:
got it!

Reply:
done